app/webpacker/css/admin/products_v3.scss

```diff
@@ -115,8 +114,12 @@
   }
 
   #filters {
-    gap: 20px;
-    align-items: flex-end;
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: 1fr;
+    grid-column-gap: 20px;
+    align-items: end;
+    margin-bottom: 20px;
```

Comment:
I think we might have had this conversation before, but I can't remember:

I find the grid syntax not as clear for what it is doing. I think I prefer the old style of using classes like `.two.columns` , but admit that is limited. This is probably good, I just need to get more familiar with it :)